code/defines/procs/statistics.dm

```diff
@@ -163,4 +159,5 @@ proc/sql_commit_feedback()
 			var/DBQuery/query = dbcon.NewQuery("INSERT INTO [format_table_name("feedback")] (id, roundid, time, variable, value) VALUES (null, [newroundid], Now(), '[variable]', '[value]')")
 			if(!query.Execute())
 				var/err = query.ErrorMsg()
-				log_game("SQL ERROR during death reporting. Error : \[[err]\]\n")
\ No newline at end of file
+				log_game("SQL ERROR during feedback reporting. Error : \[[err]\]\n")
+				
```

Comment:
end of file with tabs
should just delete the tabs or make it end on the `log_game` line